### ISSUE TITLE
Deprecate in favor of NAJ?

### ISSUE THREAD

**Issue by itegulov** [near/near-abi-client-js#13]
Should this be a library of its own or should we just focus on adding support in NAJ (work started in https://github.com/near/near-api-js/pull/1045)?

I personally can't find a reason for keeping this library, but I could be missing something

CC @miraclx

**Comment by miraclx** [comment]
Yeah, I think it's reasonable to assert that consumers of this client will already be using api-js. This is a direction I'm okay with.
